tools/pontoon/users.md

```diff
@@ -0,0 +1,65 @@
+
+To create an account on [Pontoon](https://pontoon.mozilla.org/), simply click on the menu icon in the top right corner and select *Sign In*.
+
+![Pontoon login](/assets/images/pontoon/users/menu_login.png)
+
+## User settings
+
+[User settings](https://pontoon.mozilla.org/settings/) can be accessed by clicking the user avatar in the top right corner, and selecting *Settings*.
+
+![Pontoon login](/assets/images/pontoon/users/menu_settings.png)
+
+Profile picture can be set on [Gravatar](http://gravatar.com/), while the displayed name can be edited simply by clicking on it.
+
+![Pontoon login](/assets/images/pontoon/users/profile_page.png)
+
+Both *Quality checks* and *Make suggestions* are checkboxes. In the picture above *Quality checks* is disabled while *Make suggestions* is enabled.
+* *Quality checks* (check for number of sentences, placeholders, HTML tags, punctuation, etc.) are performed when saving a translation.
+* *Make suggestions* allows a translator or a manager to submit suggestions instead of translations.
+
+Both settings can also be changed directly from the gear menu (highlighted in orange) when translating strings. User settings are also accessible from this menu by selecting *Change all settings*.
+
+![Pontoon login](/assets/images/pontoon/users/translation_gear.png)
+
+*Preferred locales* can be used to display a specific group of locales before others in the *LOCALES* tab when translating strings (highlighted in red). In this case, the user chose to display *fr* and *es-ES* before other translations. Since they’re both available the tab displays 2+72, meaning 2 preferred locales plus all others.
+
+![Pontoon login](/assets/images/pontoon/users/translation_locales.png)
+
+## User roles
+
+There are currently four user roles in Pontoon.
+
+* **Contributors** can only submit suggestions. This is the default role for new users.
+* **Translators** can submit translations directly in a specific locale, and review other contributors’ suggestions.
+* **Managers** can perform the same actions as translators, but also manage other users and update their permissions.
+* **Admins** can administrate Pontoon, e.g. adding or removing projects, and act as managers for all locales.
+
+## Managing permissions
+
+A **manager** can upgrade other users’ permissions within a locale. To manage users, open the team page and select the *CONTRIBUTORS* tab (it will only be visible to **managers** and **admins**).
+
+By default there’s only a **General** section: permissions defined here will apply to all projects, but can be overridden by custom project permissions.
+
+![Pontoon login](/assets/images/pontoon/users/permissions_general.png)
+
+To move a user to a different column simply hover the username: arrows will appear to move the username on the left or right. Note that a user needs to log in at least once in Pontoon in order to upgrade their permission – simply having a Firefox Account is not sufficient.
```

Comment:
Ah, I understood your previous comment the other way around…
https://github.com/mozilla-l10n/localizer-documentation/pull/38#discussion_r126246931